Running tests on Travis results in "Cannot find module 'broccoli-funnel'"

Since `2.7.0` tests work fine locally, but Travis fails. I'm using ember-try to test release, beta and canary versions. All versions fail and travis sais `Error: Cannot find module 'broccoli-funnel'`.

Could it be that `broccoli-funnel` needs to be specified under `dependencies` instead of `devDependencies`?

--- a/package.json
+++ b/package.json
@@ -26,8 +26,6 @@
   "license": "MIT",
   "devDependencies": {
     "broccoli-asset-rev": "^2.1.2",
-    "broccoli-funnel": "^1.0.1",
-    "broccoli-merge-trees": "^1.1.1",
     "ember-cli": "1.13.8",
     "ember-cli-app-version": "0.5.0",
     "ember-cli-content-security-policy": "0.4.0",
@@ -41,8 +39,7 @@
     "ember-disable-prototype-extensions": "^1.0.0",
     "ember-disable-proxy-controllers": "^1.0.0",
     "ember-export-application-global": "^1.0.3",
-    "ember-try": "0.0.6",
-    "rsvp": "^3.2.1"
+    "ember-try": "0.0.6"
   },
   "keywords": [
     "ember-addon",
@@ -52,9 +49,12 @@
     "emberjs"
   ],
   "dependencies": {
+    "broccoli-funnel": "^1.0.1",
+    "broccoli-merge-trees": "^1.1.1",
     "ember-cli-babel": "^5.1.3",
     "ember-cli-htmlbars": "^0.7.9",
-    "ember-cli-htmlbars-inline-precompile": "^0.3.2"
+    "ember-cli-htmlbars-inline-precompile": "^0.3.2",
+    "rsvp": "^3.2.1"
   },
   "ember-addon": {
     "configPath": "tests/dummy/config",

